packages/inter-protocol/src/provisionPoolKit.js

```diff
@@ -75,60 +73,68 @@ const FIRST_LOWER_NEAR_KEYWORD = /^[a-z][a-zA-Z0-9_$]*$/;
  * Given attenuated access to the funding purse, handle requests to provision
  * smart wallets.
  *
- * @param {(depositBank: ERef<Bank>) => Promise<void>} sendInitialPayment
- * @param {() => void} onProvisioned
+ * @param {import('@agoric/zone').Zone} zone
  */
-export const makeBridgeProvisionTool = (sendInitialPayment, onProvisioned) => {
-  /** @param {ProvisionPoolKitReferences} refs */
-  const makeBridgeHandler = ({
-    bankManager,
-    namesByAddressAdmin,
-    walletFactory,
-  }) =>
-    Far('provisioningHandler', {
-      fromBridge: async obj => {
+export const prepareBridgeProvisionTool = zone =>
+  zone.exoClass(
+    'smartWalletProvisioningHandler',
+    M.interface('ProvisionBridgeHandlerMaker', {
+      fromBridge: M.callWhen(M.record()).returns(),
+    }),
+    (bankManager, walletFactory, namesByAddressAdmin, helperFacet) => ({
```

Comment:
Renaming the `forHandler` facet to be called `helperFacet` here is confusing.
```suggestion
    (bankManager, walletFactory, namesByAddressAdmin, forHandler) => ({
```

Reply:
will do

Reply:
resolved in https://github.com/Agoric/agoric-sdk/commit/9c757093b0d5492e178362e5458e02e081cbd2b0